src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -57,7 +57,10 @@ public function validateActiveUrl($attribute, $value)
 
         if ($url = parse_url($value, PHP_URL_HOST)) {
             try {
-                return count(dns_get_record($url, DNS_A | DNS_AAAA)) > 0;
+                return (bool) dns_get_record(
```

Comment:
Please leave the count greater than zero as it was. It's not related to your fix.

Reply:
But it is wrong, `dns_get_record` may return `false` which results to exception thrown on `count(false) > 0` and also causes wrong pass on tinker:
```php
validator(['htttp://invalid/test'], ['active_url'])->passes();
// returns false on the tests which is right
// returns true on tinker which is wrong!
```

Reply:
I've added 2 assertion for this and also explained on the PR description.

Reply:
When would it return false?

Reply:
Can you provide a test case which crashes with the old code?

Reply:
on tinker `dns_get_record` failure returns false and `count(false) > 0` causes a PHP warning instead of an exception error, so the try/catch doesn't work and validation returns true instead of false, I can't write a test case because it only happens on tinker:
![image](https://user-images.githubusercontent.com/56585913/79226668-4c3f7f00-7e4e-11ea-95c5-eecb5d01fb5b.png)

Reply:
As I said, I've added [this test case](https://github.com/laravel/framework/pull/32369/files#diff-577188511192a5e66c2bd31cb84d38c2R2637) but it only fails on tinker!

Reply:
Probably because your error config is hiding warnings.